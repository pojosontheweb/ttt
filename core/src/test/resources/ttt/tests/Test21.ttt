<%@ page import="java.util.List" %>
<%!
    List<List<Number>> l;
%>
this is generic...