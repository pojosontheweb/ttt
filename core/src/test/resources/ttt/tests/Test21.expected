package ttt.tests;

import java.util.List;

public class Test21 extends com.pojosontheweb.ttt.Template {

	private final List<List<Number>> l;

	/**
	 * Creates an instance of this template.
	 *
	 */
	public Test21(List<List<Number>> l) {
		this.l = l;
	}

	@Override
	public void doRender(com.pojosontheweb.ttt.TttWriter out) throws Exception {
		write(out, "this is generic..." );
	}
}
